src/number_theory/nat_prop.lean

```diff
@@ -0,0 +1,96 @@
+/-
+Copyright (c) 2021 Ashvni Narayanan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ashvni Narayanan
+-/
+import data.nat.parity
+import ring_theory.int.basic
+
+/-!
+# Properties of natural numbers
+This file describes some properties of natural numbers.
+
+## Tags
+Naturals
+-/
+
+namespace nat
+lemma coprime_sub {n m : ℕ} (h : n.coprime m) (hn : m ≤ n) : (n - m).coprime n :=
```

Comment:
```suggestion
lemma coprime.sub {n m : ℕ} (h : n.coprime m) (hn : m ≤ n) : (n - m).coprime n :=
```
since this enables dot notation. I'd be tempted to call this something like `sub_left` or `sub_self_left` or something, though I don't have a good combination of words.